Nav.php

```diff
@@ -125,23 +129,23 @@ public function init()
     public function run()
     {
         BootstrapAsset::register($this->getView());
-        return $this->renderItems();
+        return $this->renderItems($this->items, $this->options);
     }
 
     /**
      * Renders widget items.
      */
-    public function renderItems()
+    public function renderItems($items, $options = [])
```

Comment:
It is not good to change public method signature.
It is better to extract some method like `renderItemsInternal()`, which can be used at `renderItem()` and at `renderItems()`